### ISSUE TITLE
httpS.input use port 80 as default, but default is 443

### ISSUE THREAD

**Issue by 2002demon** [savonet/liquidsoap#1127]
Hello.

My version is 1.4.1 and i have 2 bugs.

1. input.https

input.**https**("https://domain.com/path")
liquidsoap will open with 80 port
https://domain.com:80/path
and not working ...
Please set 443 as default by https urls.
```
2020/04/01 15:24:42 [https_7656:4] Connecting to <https://domain.com:80/path>...
```

2. SOCKET COMMAND

**jingle.push https://domain.com/jingle.mp3**
SSL does not work, with various domains tested.
```
2020/04/01 15:36:53 [procol.external:4] Fetching https head for https://domain.com/jingle.mp3
2020/04/01 15:36:53 [stderr:3] Certificate[1] subject=/C=US/O=Let's Encrypt/CN=Let's Encrypt Authority X3
2020/04/01 15:36:53 [stderr:3] Certificate[1] issuer =/O=Digital Signature Trust Co./CN=DST Root CA X3
2020/04/01 15:36:53 [stderr:3] SSL: rejecting connection - error=20
```

**Comment by smimram** [comment]
Point 1 should be fixed by 2348f415ea0fca6604c96e6fd42ed85379b3b1ea, leaving this bug open for point 2 (@toots should be able to handle that).

**Comment by smimram** [comment]
Actually, the second bug is #450 I believe and shall be fixed in 1.4.2.
